### PR TITLE
AUTH:DONE WITH ERROR HANDLING

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Authentication is done in VL

### CODE REVIEW

File: src/screens/auth/login/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    }
+  };
+
+  const onFocus = (type) => {
+    dispatch({ type, payload: false });
+  };
+
+  const testUserLogin = (e) => {
+    e.preventDefault();
+    // getUserAuth(
```
Comment (ritikaawasthy) [inline src/screens/auth/login/Login.jsx]: remove these comments if not needed

**Review by ritikaawasthy — COMMENTED** [review]
The file structure is maintained nicely, the code is clean as well.
Just a suggestion :
In screens you could make forms and other things as seperate components. It will be easier to reuse stuff and this way you'll have to write lesser code

File: src/App.js [hunk 2]
```diff
@@ -41,11 +44,48 @@ function App() {
...
-        <Route path='/watchlater' element={<WatchLater />} />
-        <Route path='/Profile' element={<Profile />} />
+        <Route path='/signup' element={<Signup />} />
+        <Route path='/login' element={<Login />} />
+        <Route
+          path='/playlist'
+          element={
+            <PrivateRoute>
+              <Playlist />
+            </PrivateRoute>
```
Comment (saurabhchirde) [inline src/App.js:54]: Like the way, you have created a separate component for private routes.

**Review by saurabhchirde — APPROVED** [review]
Consistent spacing throughout the code with proper naming, which is good for the code reviewer. Couldn't find any issue as your code is easy to read. 
You can proceed to merge.

**Pull request merged**